core/pyfanos.py

```diff
@@ -0,0 +1,11 @@
+import fanos
+
+from typing import List
+
+def send(sock_fd, blob):
+  # type: (int, str) -> None
+  fanos.send(sock_fd, blob)
+
+def recv(sock_fd, fd_out):
```

Comment:
we seem to use a pretty limited subset of the kwargs for `fanos.send()` and `fanos.recv()`. so, i only added the ones we need here. will require some work to fold in the others

Reply:
OK I had to remind myself why these exist!   You're referring to the `fd0` `fd1` `fd2` args in `pyext/fanos.pyi` , which is only for `send()`.

It appears the issue is that Oil is the "headless" or "FANOS" SERVER.   So it receives field descriptors.

The CLIENT would send 3 file descriptors -- stdin, stdout, stderr.  (I just hard-coded 3 to make it easy.)

So I think it's probably fine to do this incrementally, let me read over the rest ...